pkg/server/moon.go

```diff
@@ -0,0 +1,100 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/Lighty0410/ekadashi-server/pkg/mongo"
+)
+
+type MoonPhases struct {
+	Success bool                `json:"success"`
+	Err     error               `json:"error"`
+	Resp    []MoonPhaseResponse `json:"response"`
+}
+
+type MoonPhaseResponse struct {
+	Moon Moon `json:"moon"`
+	Sun  Sun  `json:"sun"`
+}
+
+type Sun struct {
+	RiseISO time.Time `json:"riseISO"`
+}
+
+type Moon struct {
+	Rise    int       `json:"rise"`
+	RiseISO time.Time `json:"riseISO"`
+	Phase   Phase     `json:"phase"`
+}
+
+type Phase struct {
+	Name string `json:"name"`
+}
+
+func getJSON(url string, target interface{}) error {
+	myClient := &http.Client{}
+	r, err := myClient.Get(url)
+	if err != nil {
+		return err
+	}
+	defer r.Body.Close()
+	return json.NewDecoder(r.Body).Decode(&target)
+}
+func (s *EkadashiServer) moonAPI(w http.ResponseWriter, _ *http.Request) {
+	url := "http://api.aerisapi.com/sunmoon/minsk,mn?from=now&to=1month&limit=31&client_id=6foK0Hivo3udMxkoWGpL5&client_secret=xrgfvYtd3x9MY0XGK4Q7NvUoIRY5W0U2WEhTOJDC"
```

Comment:
VERY BAD: do not store your secret keys and tokens in code!!! this should be changes to envs